stable/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -48,6 +48,14 @@ spec:
         {{- if .Values.vpcId }}
         - --aws-vpc-id={{ .Values.vpcId }}
         {{- end }}
+        {{- range $key, $value := .Values.extraArgs }}
+          - --{{ $key }}={{ $value }}
+        {{- end }}
+        env:
```

Comment:
check if env is non-empty
{{- if .Values.env }}

Reply:
Fixed :+1: